soroban-env-host/Cargo.toml

```diff
@@ -29,58 +29,58 @@ ed25519-dalek = {version = ">=2.0.0", features = ["rand_core"] }
 rand = "0.8.5"
 # NB: this must match the same rand_chacha version used by ed25519-dalek above
 rand_chacha = "0.3.1"
-num-traits = "0.2.17"
-num-integer = "0.1.45"
-num-derive = "0.4.1"
-backtrace = { version = "0.3.75", optional = true }
-k256 = {version = "0.13.3", default-features = false, features = ["ecdsa", "arithmetic"]}
+num-traits = "0.2.19"
+num-integer = "0.1.46"
+num-derive = "0.4.2"
+backtrace = { version = "0.3.76", optional = true }
+k256 = {version = "0.13.4", default-features = false, features = ["ecdsa", "arithmetic"]}
 p256 = {version = "0.13.2", default-features = false, features = ["ecdsa", "arithmetic"]}
-ecdsa = {version = "0.16.7", default-features = false}
-sec1 = {version = "0.7.2"}
-elliptic-curve ={ version = "0.13.5", default-features = false}
-generic-array ={ version = "0.14.7"}
+ecdsa = {version = "0.16.9", default-features = false}
+sec1 = {version = "0.7.3"}
+elliptic-curve ={ version = "0.13.8", default-features = false}
+generic-array ={ version = "0.14.9"}
 # NB: getrandom is a transitive dependency of k256 which we're not using directly
 # but we have to specify it here in order to enable its 'js' feature which
 # is needed to build the host for wasm (a rare but supported config).
-getrandom = { version = "0.2.11", features=["js"] }
+getrandom = { version = "0.2.16", features=["js"] }
 sha3 = "0.10.8"
 # NB: this must match the same curve25519-dalek version used by ed25519-dalek
 # above used only for calibration
 curve25519-dalek = { version = ">=4.1.3", default-features = false, features = ["digest"]}
-ark-bls12-381 = { version = "0.4.0"}
-ark-bn254 = { version = "0.4.0"}
-ark-serialize = { version = "0.4.2"}
-ark-ff = { version = "0.4.2"}
-ark-ec = { version = "0.4.2"}
+ark-bls12-381 = { version = "0.5.0"}
```

Comment:
We should be able to remove `RUSTSEC-2024-0388` from deny.toml now. I'll do this after the current ci issues are fixed in this PR.